src/components/content-tab-settings.tsx

```diff
@@ -37,6 +41,16 @@ export function ContentTabSettings( { selectedSite }: ContentTabSettingsProps )
 		? `${ selectedSite.customDomain }`
 		: `localhost:${ selectedSite.port }`;
 	const protocol = selectedSite.customDomain && selectedSite.enableHttps ? 'https' : 'http';
+
+	const handleTrustCertificate = async () => {
+		await getIpcApi().trustCertificate();
+
+		// Invalidate the query to refresh the data
+		certificateTrustApi.util.invalidateTags( [ 'CertificateTrust' ] );
+
+		await refetchCertificateTrust();
```

Comment:
I think I know what happened, yet. I need to wrap `dispatch` to invalidate tags

Reply:
Yup, that's right 👍 You need the `dispatch` call. Use the `useAppDispatch` hook from `src/stores` to get a dispatch function.

Reply:
I have tried using `dispatch` and it works. I have updated the PR.